Sming/SmingCore/Network/TelnetServer.cpp

```diff
@@ -0,0 +1,139 @@
+/*
+ * telnetServer.cpp
+ *
+ *  Created on: 18 apr. 2015
+ *      Author: Herman
+ */
+
+#include "TelnetServer.h"
+#include "TcpServer.h"
+
+TelnetServer::TelnetServer() : TcpServer()
+{
+	// TODO Auto-generated constructor stub
+
+}
+
+TelnetServer::~TelnetServer()
+{
+	// TODO Auto-generated destructor stub
+}
+
+void TelnetServer::onClient(TcpClient *client)
+{
+	debugf("TelnetServer onClient %s", client->getRemoteIp().toString().c_str() );
+	curClient = client;
+	TcpServer::onClient(client);
+	curClient->sendString("Welcome to Sming / ESP6266 Telnet\r\n");
+}
+
+void TelnetServer::onClientComplete(TcpClient& client, bool succesfull)
+{
+	curClient = nullptr;
+	debugf("TelnetServer onClientComplete %s", client.getRemoteIp().toString().c_str() );
+	TcpServer::onClientComplete(client, succesfull);
+}
+
+void TelnetServer::wrchar(char c)
+{
+	char ca[2];
+	ca[0]= c;
+	curClient->write(ca,1);
+}
+
+bool TelnetServer::onClientReceive (TcpClient& client, char *data, int size)
+{
+	debugf("TelnetServer onClientReceive : %s, %d bytes \r\n",client.getRemoteIp().toString().c_str(),size );
+	debugf("Data : %s", data);
+
+	for (int i=0;i<size;i++)
+	{
+		if (data[i] >= 240) // part of Telnet command
```

Comment:
Can you discover, why 240? Also, I think, will be good move it to some define with clear naming.